scraper/README.md

```diff
@@ -89,11 +89,16 @@ This can be done with the following command:
 npm run merge:datasets
 ```
 
+It iteratively gets the German common names from https://www.wikidata.org to prevent 429 errors by sending too many requests in a short time.
```

Comment:
```suggestion
Iteratively gets the German common names from https://www.wikidata.org to prevent 429 errors by sending too many requests in a short time.
```